src/cmipcite/citations.py

```diff
@@ -134,8 +148,11 @@ def get_citations(
 
     Parameters
     ----------
-    tracking_ids_or_paths
-        Tracking IDs or paths for which to get citations
+    ids_or_paths
+        Tracking IDs PID or paths for which to get citations.
+        Tracking ids identify files. They are found in the tracking_id attribute.
+        PIDs identify datasets (a grouping of files).
+        Paths should point to a CMIP file with a tracking_id attribute.
```

Comment:
Nice. However you end up explaining this in the notebook, you could just copy that here too